app/controllers/katello/api/v2/host_errata_controller.rb

```diff
@@ -135,5 +147,17 @@ def find_errata_ids
         @errata_ids = find_bulk_errata_ids([@host], params[:bulk_errata_ids])
       end
     end
+
+    def validate_index_params!
+      if (params[:content_view_id] && params[:environment_id].nil?) || (params[:environment_id] && params[:content_view_id].nil?)
+        fail _("Either both parameters 'content_view_id' and 'environment_id' should be specified or neither should be specified")
+      end
+      if params[:type].present?
+        fail _("Type must be one of: %s" % TYPES_FROM_PARAMS.keys.join(', ')) unless TYPES_FROM_PARAMS.key?(params[:type].to_sym)
```

Comment:
Why do you need TYPES_FROM_PARAMS. You can do what you did for severity below right?

Reply:
This is validating the params, which should only be `bugfix`, `security`, or `enhancement`.  If I just did `Katello::Erratum::TYPES`, it would also allow you to send params of `optional` or `recommended`, which I don't want.